### ISSUE TITLE
Support exporting media to other locations

### ISSUE THREAD

**Issue by aarushchopra** [AFathi/ARVideoKit#12]
The [exporting methods](https://github.com/AFathi/ARVideoKit/wiki/RecordAR#exporting-methods) section outlines how to export media to a user’s Photos library. However, there are use cases where persisting media to other locations is preferred. For example, if a user wants to share a recently exported media, but does not want the media persisted to Photos, the device's temporary directory might be more suitable.

**Comment by AFathi** [comment]
@aarushchopra, 
`ARVideoKit` already exports the videos recorded to a temporary directory on the device and it returns the directory path in both the protocol method and the completion handlers of [`stop`](https://github.com/AFathi/ARVideoKit/wiki/RecordAR#func-stop_-finished_-videopath-url---swiftvoid--nil) and [`stopAndExport`](https://github.com/AFathi/ARVideoKit/wiki/RecordAR#func-stopandexport_-finished-_-videopath-url-_-permissionstatusphauthorizationstatus-_-exportedbool---swiftvoid--nil).

In order to retrieve the local path, do the following steps:
1. Disable [`deleteCacheWhenExported`](https://github.com/AFathi/ARVideoKit/wiki/RecordAR#var-deletecachewhenexportedbool)
```
recorder?.deleteCacheWhenExported = false
```

2. Retrieve the local file path using [`stop`](https://github.com/AFathi/ARVideoKit/wiki/RecordAR#func-stop_-finished_-videopath-url---swiftvoid--nil)
```
recorder?.stop() { path in
//use the file path to export or preview inside your application
print(path)
}
```

3. You may also use the protocol method [`didEndRecording`](https://github.com/AFathi/ARVideoKit/wiki/RecordARDelegate#func-recorderdidendrecording-pathurl-with-noerrorbool)

```
func recorder(didEndRecording path:URL, with noError:Bool) {
   if noError {
      //use the file path to export or preview inside your application
      print(path)
    }
}
```

Make sure to setup the `delegate` in your class to use the `didEndRecording` protocol method.

**Comment by aarushchopra** [comment]
Oh, my bad! I think I missed that because I was expecting to find it in the "Exporting Methods" section. Thanks for clarifying!

On an unrelated note, are you planning to add support for CocoaPods? Should I make another issue for that?

**Comment by AFathi** [comment]
@aarushchopra,

Yes, I'm planning on publishing `ARVideoKit` on CocoaPods, Carthage, and Swift Package Manager.
This will be available when I open source the project in early 2018!

You can open another issue for that.

-Ahmed
